examples/update-demo/4-rolling-update.sh

```diff
@@ -26,8 +26,8 @@ fi
 NEW_IMAGE=${1:-kitten}
 TIMING=${2:-10s}
 export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
-export KUBECFG=${KUBECFG-$KUBE_REPO_ROOT/cluster/kubecfg.sh}
+export KUBECTL=${KUBECTL-$KUBE_REPO_ROOT/cluster/kubectl.sh}
 
-set -x
+SCHEMA=${KUBE_REPO_ROOT}/examples/update-demo/kitten-rc.yaml
 
-$KUBECFG -image $DOCKER_HUB_USER/update-demo:$NEW_IMAGE -u $TIMING rollingupdate update-demo
+cat ${SCHEMA} | sed "s/DOCKER_HUB_USER/${DOCKER_HUB_USER}/" | ${KUBECTL} rollingupdate update-demo-nautilus -f -
```

Comment:
Added kitten|nautilus-rc to examples test, and I think it passes now. The test doesn't actually check that DOCKER_HUB_USER is a valid docker username, so the substitution shouldn't matter.